connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -1442,7 +1442,7 @@ public void testJoinLeaderCatchUpFails() throws Exception {
         // Reading to end of log times out
         configBackingStore.refresh(EasyMock.anyLong(), EasyMock.anyObject(TimeUnit.class));
         EasyMock.expectLastCall().andThrow(new TimeoutException());
-        member.maybeLeaveGroup("test join leader catch up fails");
+        member.maybeLeaveGroup(EasyMock.eq("taking too long to read the log"));
```

Comment:
Obviously a pre-existing problem, but we should probably try to avoid brittle expectations like this.

Reply:
Yeah, my other option would be to accept `any` String as reason, but I felt that the reason is connected with the actual call, so I think the assertion can be a bit specific here. I hope the reason won't change often. If and when it changes we'll have to change this test case too.

Reply:
If the reason is important, maybe we should put an enum type behind it or something. Anyway, not a major issue.